tcl/util_string.tcl

```diff
@@ -229,3 +229,66 @@ proc qc::strip_common_leading_whitespace {text} {
     }
     return $text
 }
+
+proc qc::spell_index { corpus_file } {
+    #| Takes a filename containing a text corpus & creates spell_corpus table
+    #| containing each word and its number of entries.
+    #| spell_corpus table is:
+    #|         create table spell_corpus (
+    #|                  word text,
+    #|                  entries int
+    #|                  );
+
+    # insert into temp table since ts_stat takes a sql query as parameter
+    db_dml {drop table if exists temp_corpus}
+    db_dml {create table temp_corpus(words text)}
+
+    set fh [open $corpus_file r]
```

Comment:
I'd prefer to pass in the corpus text and push responsibility for file read to the caller.